src/Illuminate/Routing/Route.php

```diff
@@ -171,7 +171,16 @@ protected function runCallable()
             $this->parametersWithoutNulls(), new ReflectionFunction($this->action['uses'])
         );
 
-        return call_user_func_array($this->action['uses'], $parameters);
+        $callable = $this->action['uses'];
+        $parameters = array_values($parameters);
```

Comment:
Note to anyone reviewing this. This will also be required to use unpacking. You cannot unpack an associative array.